flink-connectors/flink-orc/src/main/java/org/apache/flink/orc/OrcTableSource.java

```diff
@@ -213,7 +213,8 @@ public boolean isFilterPushedDown() {
 
 	@Override
 	public String explainSource() {
-		return "OrcFile[path=" + path + ", schema=" + orcSchema + ", filter=" + predicateString() + "]";
+		return "OrcFile[path=" + path + ", schema=" + orcSchema + ", filter=" + predicateString()
```

Comment:
in `predicateString` method, it's better return `TRUE` if predicates is null or empty, otherwise return predicates string

Reply:
changed accordingly. actually the check for empty was missing